DCS_STATUS/BitStatus.txt

```diff
@@ -0,0 +1,31 @@
+HV boards (CAEN A1515)
```

Comment:
is this file imported or used anywhere?
if not, its information would better be served in a user guide